apps/web/src/pages/releases/[packSlug].tsx

```diff
@@ -118,9 +118,6 @@ export default function ReleasePage({
 
 export const getServerSideProps: GetServerSideProps = async (context) => {
   const user = await getAuthenticatedUser(context)
-  if (!user) {
-    return handleUnauthenticatedRedirect(context.resolvedUrl)
-  }
```

Comment:
Looks like `handleUnauthenticatedRedirect` isn't being used but is still in this file.